### PR TITLE
Dala 4191 minor update sfdr data model

### PR DESCRIPTION
**Pull request by jhaerDfine**
# Pull Request \ Dala 4191 minor update sfdr data model
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one test exists testing the new feature
  - [x] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [x] At least 3 consecutive CI runs are successfully executed to ensure that there are no flaky tests.
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [x] IF there are changes done to the framework data models or to a database entity class, the following steps were completed in order
  - [x] A fresh clone of dataland.com is generated (see Wiki page on "OTC" for details)
  - [x] The feature branch is deployed to clone with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on the clone server. Testing of the new feature should (also) be done by a second, independent reviewer from the dev team
  - [x] The feature branch is deployed to dev1 with `Reset non-user related Docker Volumes & Re-populate` turned on, and it's verified that the CD run is green  
- [ ] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [ ] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [ ] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on dev server. Testing of the new feature should (also) be done by a second, independent reviewer from the dev team

### CODE REVIEW

File: dataland-frontend/tests/e2e/fixtures/common/YesNoFixtures.ts [hunk 1]
```diff
@@ -16,3 +16,11 @@ export function generateYesNo(): YesNo {
 export function generateYesNoNa(): YesNoNa {
   return pickOneElement(Object.values(YesNoNa));
 }
+
+/**
+ * Randomly returns Yes, No or Na
```
Comment (aliabdoli77) [inline dataland-frontend/tests/e2e/fixtures/common/YesNoFixtures.ts:21]: doc anpassen

File: dataland-frontend/tests/component/components/forms/parts/fields/CurrencyDataPointFormField.cy.ts [hunk 2]
```diff
@@ -9,22 +9,22 @@ describe("Component test for CurrencyDataPointFormField", () => {
...
       .should("not.have.class", "p-disabled")
       .find("span")
-      .should("not.have.text", "NA");
-    cy.get('div[data-test="dataQuality"] .form-field-label span.asterisk').should("exist");
+      .should("have.text", "Data quality");
+    cy.get('div[data-test="dataQuality"] .form-field-label span.asterisk').should("not.exist");
   }
 
-  it("Quality field should be NA if the value field has no value and currency field should work as expected", () => {
+  it("Quality field should be Data quality if the value field has no value and currency field should work as expected", () => {
```
Comment (aliabdoli77) [inline dataland-frontend/tests/component/components/forms/parts/fields/CurrencyDataPointFormField.cy.ts:27]: can't this test just be removed?

Comment (jhaerDfine) [reply dataland-frontend/tests/component/components/forms/parts/fields/CurrencyDataPointFormField.cy.ts]: its a test that verifies that you cannot upload a quality value if you do not put a value

Comment (aliabdoli77) [reply dataland-frontend/tests/component/components/forms/parts/fields/CurrencyDataPointFormField.cy.ts]: I see know 👍

File: dataland-frontend/tests/e2e/fixtures/common/YesNoFixtures.ts (hunk 1 above)
Comment (jhaerDfine) [reply dataland-frontend/tests/e2e/fixtures/common/YesNoFixtures.ts:21]: will be pushed soon

**Pull request merged**